starknet_py/net/models/transaction_test.py

```diff
@@ -36,22 +29,3 @@ def test_declare_compress_program(balance_contract):
 
     deserialized = cast(Declare, schema.load(serialized))
     assert deserialized.contract_class == contract_class
-
-
-def test_serialize_deserialize_invoke():
-    data = {
-        "sender_address": "0x1",
-        "calldata": ["0x1", "0x2", "0x3"],
-        "max_fee": "0x1",
-        "signature": [],
-        "nonce": "0x1",
-        "version": "0x1",
-        "type": "INVOKE_FUNCTION",
-    }
-    invoke = InvokeV1Schema().load(data)
-    serialized_invoke = InvokeV1Schema().dump(invoke)
-
-    assert isinstance(invoke, InvokeV1)
-    assert invoke.type == TransactionType.INVOKE
-    assert isinstance(serialized_invoke, dict)
-    assert serialized_invoke["type"] == "INVOKE_FUNCTION"
```

Comment:
Could you please explain why do we delete this test?

Reply:
We don't use these classes to serialize transactions, and the `type` attribute's shift to a property caused this test to fail. Generally, we should try to find a way to unify these classes with the transaction classes from the `client_models.py`.